Common/Systems/ModPlayers/ClassModPlayer.cs

```diff
@@ -29,12 +29,13 @@ public override void ModifyStartingInventory(IReadOnlyDictionary<string, List<It
 		item.TurnToAir();
 		vanillaItems.Add(item);
 
-		item = new Item();
-		item.SetDefaults(ItemID.CopperPickaxe);
```

Comment:
We don't want to remove the copper pickaxe, otherwise everything else looks fine.

Just gotta add that back in and this looks good. Ty

Reply:
Got it, added it back again.